docs/release-notes/8.2.asciidoc

```diff
@@ -41,6 +41,7 @@
 ==== Known issues
 * On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.
 * Indicator match rules cannot use the `.items-*` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules ({pull}133457[#133457]).
+* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting the unsupported exception and refreshing the rule ({issue}136340[#136340]).
```

Comment:
```suggestion
* The `matches` operator in the *Add Rule Exception* flyout does not work because wildcard matches are not supported for rule exceptions. Using the `matches` operator will cause rule exceptions and their associated rules to fail. You can restore failed rules by deleting unsupported exceptions and refreshing the rules ({issue}136340[#136340]).
```